src/timestamps.ts

```diff
@@ -0,0 +1,12 @@
+import * as t from "io-ts";
+import { UTCISODateFromString } from "@pagopa/ts-commons/lib/dates";
+
+export const Timestamps = t.type({
+  createdAt: UTCISODateFromString,
+  updatedAt: UTCISODateFromString,
+});
+
+export const timestamps = () => ({
+  createdAt: new Date(),
+  updatedAt: new Date(),
```

Comment:
If you don't need the full date object `Date.now()` has better perfomance

Reply:
the full date object right now is required by `Cosmos DB` for serialization purposes